_pages/how-we-work/sensitive-information.md

```diff
@@ -10,6 +10,9 @@ Here's what you need to know about sensitive information at TTS.
 
 Anything that would make our systems vulnerable or would impact the privacy of others if it fell into the wrong hands. To learn what information we consider sensitive, see [our Open Source Policy practices guide](https://github.com/18F/open-source-policy/blob/master/practice.md#protecting-sensitive-information). See also: [the GSA Controlled Unclassified Information (CUI) Guide](https://insite.gsa.gov/employee-resources/information-technology/security-and-privacy/controlled-unclassified-information-cui/cui-guide).
 
+## What to do if you find or expose sensitive information
+Immediately notify us in Slack channel [#incident-response](https://gsa-tts.slack.com/archives/C0G6G1UNM) and by email [gsa-ir@gsa.gov](mailto:gsa-ir@gsa.gov). Also review our [vulnerability disclosure policy and procedures](../responding-to-public-disclosure-vulnerabilities/)
```

Comment:
Good idea to mention they should disclose.

- Thoughts on using an [alert component](https://designsystem.digital.gov/components/alert/) near the top of the page, so it stands out?
- Mind if we link to the [security incidents](https://handbook.tts.gsa.gov/security-incidents/#reporting-other-incidents) page instead, to keep the information centralized?

Reply:
The alert feels like overkill, I like @JJediny's update, at least enough to merge this :)